modules/core/04-channel/keeper/upgrade.go

```diff
@@ -56,6 +57,34 @@ func (k Keeper) WriteUpgradeInitChannel(ctx sdk.Context, portID, channelID strin
 	emitChannelUpgradeInitEvent(ctx, portID, channelID, currentChannel, upgrade)
 }
 
+// upgradeTry
```

Comment:
nit: godoc needs updating, but assuming this is already done in the other PR... feel free to ignore

Reply:
yea, i will update this in the last handler logic PR 👍